packages/web-components/src/components/feature-card-block-medium/feature-card-block-medium.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { customElement, html, LitElement } from 'lit-element';
+import settings from 'carbon-components/es/globals/js/settings';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
+import styles from './feature-card-block-medium.scss';
+
+const { prefix } = settings;
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * Feature Card Block Medium
+ *
+ * @element dds-feature-card-block-medium
+ */
+@customElement(`${ddsPrefix}-feature-card-block-medium`)
+class DDSFeatureCardBlockMedium extends LitElement {
+  static styles = styles;
+
+  protected render() {
+    return html`
+      <div class="${prefix}--feature-card-block-medium__container"><slot name="heading"></slot><slot></slot></div>
```

Comment:
Given `<dds-feature-card-block-medium>` itself (host element) and its corresponding style rule (with `:host(dds-feature-card-block-medium)` selector) implements the corresponding element to React `<section class="bx--feature-card-block-medium">`, and `lit-html` doesn't have an error like `Adjacent JSX elements must be wrapped in an enclosing tag`, I think below will work.

```suggestion
      <slot name="heading"></slot><slot></slot>
```

Reply:
Got it, thanks @asudoh! 👍

Reply:
Cool thanks for the update @kennylam!